app/build.gradle

```diff
@@ -16,10 +16,11 @@ android {
         targetSdkVersion 30
         versionCode 1
         versionName "1.0"
-
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        buildConfigField("String", "APPID","\"16ed760ab156214729d1e276a01e9653\"")
+        testInstrumentationRunner "androidx.test.runer.AndroidJUnitRunner"
```

Comment:
it is okay this change from runner to runer?

Reply:
Its was a typo mistake, now its no more, thx